src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -140,6 +140,8 @@ float svgUtilStrtof(const char *nPtr, char **endPtr)
         }
         val += static_cast<float>(decimalPart) / static_cast<float>(pow10);
         a = iter;
+        //skip if there is a space after the dot.
```

Comment:
we can goto immediately at line 132?

Reply:
Thank you for your comment.
I move to before decimal part divide calculation.